src/envoy/http/backend_routing/filter.cc

```diff
@@ -28,7 +28,15 @@ namespace backend_routing {
 
 namespace {
 struct RcDetailsValues {
-  const std::string BadRequest = "bad_request";
+  // The request doesn't contain PATH header.
+  const std::string MissingPath = "backend_routing_bad_request{MISSING_PATH}";
```

Comment:
Will these changed rc_detail show up in the Report call?   If so,  I don't see the change in integration tests

Reply:
There is one integration test change.

Most of them are prevented by path matcher filter.